qiskit_experiments/randomized_benchmarking/rb_utils.py

```diff
@@ -0,0 +1,262 @@
+            gatelen (float): length of the gate.
+        Returns:
+            float: coherence limited error per gate.
+        Raises:
+            ValueError: if there are invalid inputs
+        """
+        # pylint: disable = invalid-name
+
+        T1 = np.array(T1_list)
+
+        if T2_list is None:
+            T2 = 2 * T1
+        else:
+            T2 = np.array(T2_list)
+
+        if len(T1) != nQ or len(T2) != nQ:
+            raise ValueError("T1 and/or T2 not the right length")
+
+        coherence_limit_err = 0
+
+        if nQ == 1:
+
+            coherence_limit_err = 0.5 * (
+                1.0 - 2.0 / 3.0 * np.exp(-gatelen / T2[0]) - 1.0 / 3.0 * np.exp(-gatelen / T1[0])
+            )
+
+        elif nQ == 2:
+
+            T1factor = 0
+            T2factor = 0
+
+            for i in range(2):
+                T1factor += 1.0 / 15.0 * np.exp(-gatelen / T1[i])
+                T2factor += (
+                    2.0
+                    / 15.0
+                    * (
+                        np.exp(-gatelen / T2[i])
+                        + np.exp(-gatelen * (1.0 / T2[i] + 1.0 / T1[1 - i]))
+                    )
+                )
+
+            T1factor += 1.0 / 15.0 * np.exp(-gatelen * np.sum(1 / T1))
+            T2factor += 4.0 / 15.0 * np.exp(-gatelen * np.sum(1 / T2))
+
+            coherence_limit_err = 0.75 * (1.0 - T1factor - T2factor)
+
+        else:
+            raise ValueError("Not a valid number of qubits")
+
+        return coherence_limit_err
+
+    @staticmethod
+    def calculate_1q_epg(
+        epc_1_qubit: float,
+        qubits: Iterable[int],
+        backend: Backend,
+        gates_per_clifford: Dict[Tuple[Iterable[int], str], float],
+    ) -> Dict[int, Dict[str, float]]:
+        r"""
+        Convert error per Clifford (EPC) into error per gates (EPGs) of single qubit basis gates.
+        Args:
+            epc_1_qubit: The error per clifford rate obtained via experiment
+            qubits: The qubits for which to compute epg
+            backend: The backend on which the experiment was ran
+            gates_per_clifford: The computed gates per clifford data
+        Returns:
+            A dictionary of the form (qubits, gate) -> value where value
+            is the epg for the given gate on the specified qubits
+        """
+        error_dict = RBUtils.get_error_dict_from_backend(backend, qubits)
+        epg = {qubit: {} for qubit in qubits}
+        for qubit in qubits:
+            error_sum = 0
+            found_gates = []
+            for (key, value) in error_dict.items():
+                qubits, gate = key
+                if len(qubits) == 1 and qubits[0] == qubit and key in gates_per_clifford:
+                    found_gates.append(gate)
+                    error_sum += gates_per_clifford[key] * value
+            for gate in found_gates:
+                epg[qubit][gate] = (error_dict[((qubit,), gate)] * epc_1_qubit) / error_sum
```

Comment:
It would be better to keep sigma.

Reply:
Not sure I follow you - sigma instead of what?

Reply:
Tough EPC has error (sigma=stdev), this information is not used by EPG. Perhaps this is future extension. Can you write an issue for remider?

Reply:
Since EPG is calculated from EPC, it should be possible to calculate the error bounds for EPG.